ngraph/python/tests/test_onnx/test_zoo_models.py

```diff
@@ -559,7 +559,8 @@
         "url": _GITHUB_MODELS_LTS + "vision/classification/zfnet-512/model/zfnet512-8.tar.gz",
     },
     {
-        "model_name": "zfnet512_opset9",
+        "model_name": ""
+                      "",
```

Comment:
I still see empty model name.

Reply:
Yep, sorry, I commited  this change but didn't push to upstream